packages/angular/src/stories/badge/badge.stories.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc.
+ * All Rights ReserveThis software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { moduleMetadata } from '@storybook/angular';
+import { withDesign } from 'storybook-addon-designs';
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+const basicTemplate = require('!!raw-loader!./basic.html');
+
+export default {
+  title: 'Badge',
+  decorators: [
+    withDesign,
+    moduleMetadata({
+      schemas: [CUSTOM_ELEMENTS_SCHEMA],
```

Comment:
`CUSTOM_ELEMENTS_SCHEMA` needed for `clr-icon`?

Reply:
Nope, I think I got the @clr/icons working with a normal import and this can be removed.